### PR TITLE
[HSDK-387] Added documentation for Huma Caching module

### PR DESCRIPTION
**Pull request by EvgeniyDavydenko**

### CODE REVIEW

File: sdk/Caching/Android.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+```
+2. Initialize `HumaCachingManager` in your Application class:
+```kotlin
+HumaCachingManager.init()
+```
+
+</TabItem>
+<TabItem value="readme" label="Readme" default>
+
+Huma Android SKD uses Room library which provides an abstraction layer over SQLite to allow fluent database access for persisting data locally.
```
Comment (valeriy-huma) [inline sdk/Caching/Android.mdx:28]: SDK*

File: sdk/Caching/Android.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+2. Initialize `HumaCachingManager` in your Application class:
+```kotlin
+HumaCachingManager.init()
+```
+
+</TabItem>
+<TabItem value="readme" label="Readme" default>
+
+Huma Android SKD uses Room library which provides an abstraction layer over SQLite to allow fluent database access for persisting data locally.    
+HumaCachingManager is a provider of an interfaces for interaction with local database.
```
Comment (valeriy-huma) [inline sdk/Caching/Android.mdx:29]: interface*

File: sdk/Caching/Android.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+```kotlin
+HumaCachingManager.init()
+```
+
+</TabItem>
+<TabItem value="readme" label="Readme" default>
+
+Huma Android SKD uses Room library which provides an abstraction layer over SQLite to allow fluent database access for persisting data locally.    
+HumaCachingManager is a provider of an interfaces for interaction with local database.  
+HumaCachingManager module encapsulates logic of creating instance of `Humadatabase` inheritanced from `RoomDatabase` and defines interfaces for creating DAOs classes and entities converters.
```
Comment (valeriy-huma) [inline sdk/Caching/Android.mdx:30]: inherited*

**Review by valeriy-huma — CHANGES_REQUESTED** [review]

File: sdk/Caching/Android.mdx (hunk 1 above)
Comment (EvgeniyDavydenko) [reply sdk/Caching/Android.mdx]: fixed

File: sdk/Caching/Android.mdx (hunk 2 above)
Comment (EvgeniyDavydenko) [reply sdk/Caching/Android.mdx]: fixed

File: sdk/Caching/Android.mdx (hunk 3 above)
Comment (EvgeniyDavydenko) [reply sdk/Caching/Android.mdx]: fixed

**Review by fernandod-huma — CHANGES_REQUESTED** [review]

**Review by valeriy-huma — APPROVED** [review]

File: sdk/Caching/Android.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (EvgeniyDavydenko) [reply sdk/Caching/Android.mdx]: fixed

**Review by andriib-huma — APPROVED** [review]

**Review by andriib-huma — APPROVED** [review]

**Review by fernandod-huma — APPROVED** [review]

**Pull request merged**